tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs

```diff
@@ -1689,10 +1687,10 @@ private Tournament MakeTestTournamentWithTenDivisions(int tournamentId)
             for (int i = 0; i < 10; i++)
             {
                 tournament.Divisions.Add(new Division());
-                tournament.Divisions[i].Groups = new List<Group>();
+                tournament.Divisions.ElementAt(i).Groups = new List<Group>();
                 for (int j = 0; j < 10; j++)
                 {
-                    tournament.Divisions[i].Groups.Add(new Group());
+                    tournament.Divisions.ElementAt(i).Groups.Add(new Group());
```

Comment:
Obtain reference to i-th element just once

Reply:
?

Reply:
How many traversal this methods performs over `tournament.Divisions` collection?

Reply:
10 times per division

Reply:
To reach i-th element it has to go through each element before i. This method can be simplified to obtain reference once and then use it instead of looking for it each iteration

Reply:
ok